python/paddle/v2/framework/tests/test_rank_loss_op.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+import numpy as np
+from op_test import OpTest
+
+
+class TestReshapeOp(OpTest):
+    def setUp(self):
+        self.op_type = "rank_loss"
+        num = 5
+        # P = {0, 1.0} or {0, 0.5, 1.0}
+        P = np.random.randint(0, 2, size=(num, num)).astype("float32")
+        Oi = np.random.random((num, num)).astype("float32")
+        Oj = np.random.random((num, num)).astype("float32")
+        O = Oi - Oj
+        Out = np.log(1.0 + np.exp(O)) - P * O
+        self.inputs = {'P': P, 'Oi': Oi, 'Oj': Oj}
+        self.outputs = {'Out': Out}
+
+    def test_check_output(self):
+        self.check_output()
+
+    def test_check_grad(self):
```

Comment:
Add some `check_grad_ignore_XXX` tests if posiible.
In `check_grad_ignore_XXX` tests, ignored variables' gradients will be set `nullptr` and your kernel should not compute it. 

Example: https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/framework/tests/test_mul_op.py#L21

Reply:
Done